prow/config/config_test.go

```diff
@@ -67,6 +71,33 @@ func replace(j *Presubmit, ks *Presubmit) error {
 				j.Spec.Containers[0].Args[i] = "--upload=gs://kubernetes-security-jenkins/pr-logs"
 			}
 		}
+		j.Spec.Containers[0].Args = append(j.Spec.Containers[0].Args, "--ssh=/etc/ssh-security/ssh-security")
+		sort.Strings(j.Spec.Containers[0].Args)
+		sort.Strings(ks.Spec.Containers[0].Args)
+		j.Spec.Containers[0].VolumeMounts = append(
+			[]kube.VolumeMount{
+				{
+					Name:      "ssh-security",
+					MountPath: "/etc/ssh-security",
+				},
+			},
+			j.Spec.Containers[0].VolumeMounts...,
+		)
```

Comment:
The args to `append` look backwards. The slice comes first then the element unless you are trying to alloc a new array.

Reply:
This is appending the original volume mounts to a new one, because DeepEqual is order sensitive on slices.

Reply:
(and it is valid to do this)

Reply:
@cjwagner because in the config I inserted all the volume & volume mounts to the front :jack_o_lantern: (it's going to be temporary)